src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/ajax/AjaxCreateItemResponse.java

```diff
@@ -0,0 +1,17 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.ajax;
+
+/**
+ * UI Response for successfully creating an item.  Since all items in IRIDA require an id,
+ * the identifier is returned to the client on successful creation.
+ */
+public class AjaxCreateItemResponse extends AjaxResponse {
```

Comment:
Can we change this to AjaxCreateItemSuccessResponse? Just so we stay consistent as we have the AjaxErrorResponse